---a/net/minecraft/item/ItemBucketMilk.java
+++b/net/minecraft/item/ItemBucketMilk.java
@@ -6,25 +6,22 @@
 import net.minecraft.stats.StatList;
 import net.minecraft.world.World;
 
-public class ItemBucketMilk extends Item
-{
+public class ItemBucketMilk extends Item {
+
     private static final String __OBFID = "CL_00000048";
 
-    public ItemBucketMilk()
-    {
+    public ItemBucketMilk() {
         this.setMaxStackSize(1);
         this.setCreativeTab(CreativeTabs.tabMisc);
     }
 
-    public ItemStack onItemUseFinish(ItemStack stack, World worldIn, EntityPlayer playerIn)
-    {
-        if (!playerIn.capabilities.isCreativeMode)
-        {
+    public ItemStack onItemUseFinish(ItemStack stack, World worldIn, EntityPlayer playerIn) {
+        if (!playerIn.capabilities.isCreativeMode) {
             --stack.stackSize;
         }
 
-        if (!worldIn.isRemote)
-        {
+        if (!worldIn.isRemote) {
             playerIn.clearActivePotions();
         }
 
@@ -32,20 +32,16 @@
         return stack.stackSize <= 0 ? new ItemStack(Items.bucket) : stack;
     }
 
-    public int getMaxItemUseDuration(ItemStack stack)
-    {
+    public int getMaxItemUseDuration(ItemStack stack) {
         return 32;
     }
 
-    public EnumAction getItemUseAction(ItemStack stack)
-    {
+    public EnumAction getItemUseAction(ItemStack stack) {
         return EnumAction.DRINK;
     }
 
-    public ItemStack onItemRightClick(ItemStack itemStackIn, World worldIn, EntityPlayer playerIn)
-    {
+    public ItemStack onItemRightClick(ItemStack itemStackIn, World worldIn, EntityPlayer playerIn) {
         playerIn.setItemInUse(itemStackIn, this.getMaxItemUseDuration(itemStackIn));
         return itemStackIn;
     }
-}
+}
